src/components/AddItemForm.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+const AddItemForm = ({ handleChange, handleSubmit, inputs }) => {
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <h3 htmlFor="options">Item Name:</h3>
```

Comment:
Is `htmlFor="options"` necessary if you don't have an html element with `id="options"`?